src/gerbil/prelude/core.ss

```diff
@@ -626,7 +630,9 @@ package: gerbil
         ((_ message detail ...)
          (stx-string? #'message)
          (apply raise-syntax-error #f (stx-e #'message) stx
-                (syntax->list #'(detail ...)))))))
+                (syntax->list #'(detail ...))))))
+
+    (defrules defmutable () ((_ var value) (begin (def var value) (set! var var) (void)))))
```

Comment:
can you multiline this?

Also, just `(def var #!void)` for the initial value, no need to do the computation twice.

Reply:
Computation only done once: it's `(set! var var)` not `(set! var value)`.

Reply:
ah, set! var var, missed that.
Ok, that's fine, even if a little funky :)

Reply:
Also `#!void` might upset the type analysis in a way `(set! var var)` won't.